### PR TITLE
Feature/awesome feature

### PR DESCRIPTION
**Pull request by mellisdesigns**
#### Summary
Awesome

### CODE REVIEW

File: tests/test.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+app.tests.extend({
+  beforeRun() {
+    do this
+  },
+  test('my cool test', function() {
+    do this
+  }),
+})
```
Comment (raghavendra-udyavar) [inline tests/test.js]: Cant find a semi-colon at end, which I see is added for Model.extend

File: model/model.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Model.extend({
+  name: attr('string'),
+  age: attr('number'),
+});
```
Comment (raghavendra-udyavar) [inline model/model.js:4]: there is a semi-colon here

File: controller/controller.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+Controller.extend({
+  changeAge(model, age) {
+    model.set('age', age);
```
Comment (larryPersonal) [inline controller/controller.js]: coding convention we use here is i_ for number based. 
We also should not name the storage val and input the same. 
model.set('i_age', age);

**Review by larryPersonal — CHANGES_REQUESTED** [review]

**Review by raghavendra-udyavar — CHANGES_REQUESTED** [review]
Can we have a consistency in using semi-colon

**Review by raghavendra-udyavar — APPROVED** [review]

**Review by larryPersonal — APPROVED** [review]

**Pull request merged**